docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,92 @@
+=================
+Rsync distributor
+=================
+
+Purpose:
+========
+Rsync distributor should be used for syncing already published files into remote
+machine. That means, rsync distributor isn't designed to work as standalone
+distributor.
+
+How it works
+============
+Rsync distributor publishes in fast-forward mode by default. It selects
+new content since last_published timestamp, however it also check predistributor
+publish history if there weren't publishes that were non-fast-forward or
+failed. In that case, cdn_distributor publishes everything from scratch.
+Publish process is diveded into multiple steps:
+unit query, origin publish, content publish, extra data publish
+
+1. Unit query step fetchs units from database and creates realive symlinks to
+units _storage_dir in <working_repo_dir>/<extra_dir>/.relative
+
+2. origin publish syncs hard files from /var/lib/pulp/content/ to remote server
+There can by multiple origin publish steps if there are more then one content
+types to be synced.
+
+3. content publish syncs files <working_repo_dir>/<extra_dir>/.relative to
+remote server. Remote repo directory is calculated from ``remote_root`` and
+relative_url.
+
+4. extra data publish happens only if there are extra data to be synced. For
+example extra data for rpm repo are repodata/* files.
+
+
+Supported types
+==============
+For now, rsync distributor can sync any content from rpm and docker repositories
```

Comment:
If this supports RPM and docker, it should live in Pulp platform instead of here in the RPM plugins.